csatar-plugins/forms/components/BasicForm.php

```diff
@@ -87,36 +154,70 @@ public function getForms() {
      */
     public function onRun() {
 
-        // Run application
-
-
         // Handle file upload requests
         if ($handler = $this->processUploads()) {
             return $handler;
         }
 
+        $this->getComponentSettings();
+
         // Render frontend
         $this->addCss('/modules/system/assets/ui/storm.css');
         $this->addJs('/modules/system/assets/ui/storm-min.js');
         $this->addJs('/plugins/csatar/forms/assets/vendor/dropzone/dropzone.js');
         $this->addJs('/plugins/csatar/forms/assets/js/uploader.js');
         $this->addJs('/plugins/csatar/forms/assets/js/positionValidationTags.js');
 
-        $model_id = $this->param('model_id', 'new');
-        $this->renderedComponent = $this->createForm($this->getForm(), $model_id);
+        $form = $this->getForm();
+        $this->recordKeyValue = $this->param($this->recordKeyParam);
```

Comment:
Nincs egy $this->params, ami sorrendben tartalmazza a paramétereket, s akko az elsőnek a neve lenne a recordKeyValue?